recipe/meta.yaml

```diff
@@ -25,24 +25,41 @@ build:
 requirements:
   build:
     - r-base
+    - r-r6
+    - r-assertthat
+    - r-crayon
+    - r-debugme
+    - r-testthat
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
 
   run:
     - r-base
+    - r-r6
+    - r-assertthat
+    - r-crayon
+    - r-debugme
+    - r-testthat
+    - libgcc  # [not win]
 
 test:
   commands:
     - $R -e "library('callr')"  # [not win]
     - "\"%R%\" -e \"library('callr')\""  # [win]
 
 about:
-  home: https://github.com/MangoTheCat/callr
+  home: https://cran.r-project.org/package=callr
+  dev_url: https://github.com/r-lib/callr
   license: MIT
   summary: It is sometimes useful to perform a computation in a separate R process, without affecting
     the current R process at all. This packages does exactly that.
   license_family: MIT
+  license_file: LICENSE
```

Comment:
@rvalieris Could you please remove the `license_file` entry. The file `LICENSE` shipped with R packages licensed under MIT is just a stub, not a full license: https://cran.r-project.org/web/packages/callr/LICENSE

Reply:
good catch, didnt notice that.